### PR TITLE
TIED-71

### PR DESCRIPTION
**Pull request by arpargo**
## Description

Adding simple checks for components to see that they get rendered without crashing.

## Related Issue(s)

**[TIED-71](https://helsinkisolutionoffice.atlassian.net/browse/TIED-71)**

## Motivation and Context

To enable the service move to Platta, we need to increase our code coverage and starting by adding some baseline tests is the way to go.

## How Has This Been Tested?
By running tests locally.

## Screenshots / Videos

[TIED-71]: https://helsinkisolutionoffice.atlassian.net/browse/TIED-71?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/Bulk/BulkView/__tests__/BulkView.spec.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                fetchBulkUpdate={dummyFunction}
+                fetchNavigation={dummyFunction}
+                getAttributeName={dummyFunction}
+                // isFetchingNavigation: PropTypes.bool,
+                // isUpdating: PropTypes.bool,
+                items={[]}
+                // itemsIncludeRelated: PropTypes.bool,
+                match={{}}
+                push={dummyFunction}
+                // selectedBulk: PropTypes.object,
```
Comment (voneiden) [inline src/components/Bulk/BulkView/__tests__/BulkView.spec.js:39]: Are these comments  here intentionally or leftovers from development?

**Review by voneiden — APPROVED** [review]
LGTM, surprised the coverage didn't go up more

File: src/components/Bulk/BulkView/__tests__/BulkView.spec.js (hunk 1 above)
Comment (arpargo) [reply src/components/Bulk/BulkView/__tests__/BulkView.spec.js:39]: Heh, these were leftovers from trying to figure out truly required props, i thought i managed to get them all removed but apparently I missed this file. Fixing.

**Review by mikkojamG — COMMENTED** [review]
Nice work 👍  I had this idea that since we're trying to push using functional components instead of class based components in the future, we could start writing tests for existing functional components (like Banner.jsx) with [React Testing Library](https://testing-library.com/docs/react-testing-library/intro/). I think tests for class based components (like Conversion.jsx) could still be written with Enzyme, as it typically works a bit better with class based components. As we slowly move towards refactoring components into functional we could also replace Enzyme with Testing library. 🤔

**Pull request merged**